### ISSUE TITLE
Question about training setting.

### ISSUE THREAD

**Issue by weeknan** [pengkun-jiao/Dual-LoRA#1]
Hi, thanks for your work.
What data do you use during the two-stage fine-tuning stage mentioned in Sec 3.5? Is it the default  pre-training and fine-tuning data used in LLaVA-1.5, or the downstream data used for evaluation, eg UniFood?

**Comment by pengkun-jiao** [comment]
Thank you for your attention to our work! We used downstream data for the two-stage fine-tuning. Please note that the vision projector of the base model was previously trained on general data before being used for downstream fine-tuning.

**Comment by weeknan** [comment]
Hi, I still have some questions and would like to confirm my understanding.

For the setup in Table 1, you first initialize a pretrained LLaVA model, and then perform the following two-stage fine-tuning using the UniFood training set, with results reported on the UniFood test set:
1. Fine-tune the VCE and vision feature projector on UniFood training set;
2. Fine-tune the VCE, vision feature projector, and Dual-LoRA on UniFood training set.

For the setup in Table 5, you use LLaVA’s official pre-training and fine-tuning data to perform the following two-stage training, and report results on general-purpose benchmarks:
1. Train the VCE and vision feature projector on 558K pretrain data;
2. Train the VCE, vision feature projector, and Dual-LoRA on 665K finetune data.

Is my understanding correct? Please kindly correct me if anything is wrong; really appreciate!

**Comment by pengkun-jiao** [comment]
We use specific data for different training purposes. For the UniFood benchmark, descriptive data, including ingredient recognition and recipe generation task data, are used in the first stage (train VCE and vision projector). In the second stage, all task data are used (train VCE, vision projector, and Dual-LoRA). For the general benchmark, the training data used in each stage follow the official configuration, as you pointed out.

**Comment by weeknan** [comment]
i see. Thank you.
